CONTRIBUTING.md

```diff
@@ -1,31 +1,62 @@
 
 ### Review Process for this Repo
 
-After following the steps in the [Contribution Guide](http://docs.mattermost.com/developer/contribution-guide.html), submitted pull requests go through the review process outlined below. We aim to start reviewing pull requests in this repo the week they are submitted, but the length of time to complete the process will vary depending on the pull request.
+When you submit a pull request, it goes through the review process outlined below. We aim to start reviewing pull requests in this repo the week they are submitted, but the length of time to complete the process will vary depending on the pull request.
 
 The one exception may be around release time, where the review process may take longer as the team focuses on our [release process](https://docs.mattermost.com/process/release-process.html). 
 
-#### `Stage 1: PM Review`
+#### PR submitted
+
+After a PR is submitted, a core committer applies labels and notifies PMs that there is a PR awaiting review by posting in the [PM/Docs PR Review channel](https://pre-release.mattermost.com/core/channels/pmdocs-pr-review-pub), which is a channel to discuss community pull requests that need review by PMs.
+
+Then, one or more of the labels is applied:
+ - `Awaiting PR`: Applied if the PR is awaiting another to be merged. For example, when a client PR is awaiting a server PR to be merged first. Once the PR is no longer blocked, the core committer removes the `Awaiting PR` label
+ - `1: PM Review`: Applied if the PR has UI changes or functionality that PMs can test on test servers called "spinmints"
+ - `Setup Test Server`: Applied if the PR is queued for PM testing
+ - `Work in Progress`: Applied if the PR is unfinished and needs further work before it's ready for review
+
+#### Stage 1: PM Review
 
 A Product Manager will review the pull request to make sure it:
 
-1. Fits with our product roadmap
-2. Works as expected
-3. Meets UX guidelines
+ - Fits with our product roadmap
+ - Works as described in the ticket
+ - Meets UX guidelines
+
+This step is sometimes skipped for bugs or small improvements with a well defined ticket.
+
+When the review process begins, the PM applies a milestone:
+ - Set for next release if the PM thinks there is enough time for the PR to be merged and sufficiently tested on `master` before code complete.
+ - Set for a future release if PR is too large to test prior to the code complete date
+   - PM responds to submitter letting them know that PR may have a delay in review due to the release cycle
+
+Next, the PM tests changes on the spinmint:
+ - If changes are required, PM submits review as "Changes Requested", with a comment on the areas that require updates. Comment explains why changes are needed linking back to design principles.
+   - PM applies `Awaiting Submitter Action` label to more easily query the PR queue
```

Comment:
There are instances where a PR sits for a long time in PM review awaiting for submitter action, but without a `changes requested` applied.

This is because sharing long bits of feedback is easier through a comment than the small `changes request` comment box.

E.g. for the attached two cases, if there was no `awaiting submitter action` label, it would appear that no one has reviewed the PR in a month even though that's not the case.

![image](https://cloud.githubusercontent.com/assets/13119842/24083446/48ce7ec2-0cad-11e7-8f0a-d790ab6d32fc.png)